strapi/strapi
# enhancement: add support for user database migrations in typescript

### What does it do?

Recongnize /dist/database/migrations in ts project

### Why is it needed?

https://github.com/strapi/strapi/issues/17820

### How to test it?

Setup a typescript project with ts migrations, verify those migrations are executed.

### Related issue(s)/PR(s)

https://github.com/strapi/strapi/pull/18010
